app/admin/book.rb

```diff
@@ -4,11 +4,11 @@
     column :title
     column :author
     column :published_date
-    column :price
+    column :price_cents
     column :category
     column :description
     actions
   end
 
-  permit_params :title, :author, :published_date, :price, :category, :description
+  permit_params :title, :author, :published_date, :price_cents, :category, :description
 end
```

Comment:
Fixed a failing admin spec caused by last merge. Unrelated to this PR's purpose though.